change_quotes.py

```diff
@@ -104,6 +105,23 @@ def run_each(self, edit, sel_region):
         if not region:
             return
 
+        if self.custom:
+            fn_name = self.custom[0]
+            try:
+                fn = getattr(self, fn_name)
+            except AttributeError:
+                return
+
+            try:
+                fn_kwargs = self.custom[1]
+                if not isinstance(fn_kwargs, dict):
```

Comment:
Oh that's a good point - yeah I think I'll remove the guard for fetching the custom method, or at least show an error message.  Fine for now, anyway.